rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-utility.yml

```diff
@@ -70,3 +70,21 @@
     msg: "One or more of the cinder services are in the down state. Check the output above for more detail."
   when: "'down' in '{{ item.split(' ')[-1] }}'"
   with_items: "{{ cinder_output.stdout_lines | default([]) }}"
+
+# Adds orchestration service checks per https://rpc-openstack.atlassian.net/browse/UG-543
+- name: Grab relevant output of orchestration service list
+  shell: |
+    . ~/openrc
+    openstack orchestration service list --insecure
+  register: orchestration_output
+  failed_when: "orchestration_output.rc != 0"
+
+- name: Display output of orchestration service list
+  debug:
+    var: orchestration_output
+
+- name: Fail if any of the orchestration services are down
+  fail:
+    msg: "One or more of the orchestration services are down. Check the output above for more detail."
+  when: "'down' in '{{ item.split(' ')[-1] }}'"
```

Comment:
I think you make a fair point, any general bugs with this file are separate from the addition of this functionality. That said, the task you have added does not match the implementation of the others. For example:
```yaml
openstack compute service list -c Binary -c Host -c State -f value
```
This command limits the output to an ordered list of space-separated specific values. While I would prefer the use of JSON or YAML, I think this is preferable to the default table output. `--insecure` is not used here or by any of the other commands in the file. It looks like you've copied this task from the other file which is doing things differently.